src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -395,12 +435,12 @@ public Object onTraversalSuccess() {
      * @param ubSampleIndex  Upper bound of sample index -- inclusive (0-based)
      * @return  GenomicsDB import configuration object
      */
-    private GenomicsDBImportConfiguration.ImportConfiguration createImportConfiguration(
+    private static GenomicsDBImportConfiguration.ImportConfiguration createImportConfiguration(
         final String workspace,
         final String arrayName,
         final long variantContextBufferSize,
         final long segmentSize,
-        final long lbSampleIndex,
+        final  long lbSampleIndex,
```

Comment:
huh, i'm not sure what's going on here but I can't see the extra space in my editor, maybe intellij fixed it for me automagically